wsd/AdminModel.cpp

```diff
@@ -45,6 +45,19 @@ std::ostream& print(std::ostream& oss, const std::string_view prefix, const std:
     return oss;
 }
 
+
+std::string concat(const std::string_view &prefix, const std::string_view name,
```

Comment:
Isn't this identical to `print()`, a few lines above (using `string` instead of `ostream`) ?

If we really need a string version, we can re-use the `print()`, like this:
```
std::string concat(const std::string_view &prefix, const std::string_view name,const std::string_view suffix)
{
    std::ostringstream oss;
    print(oss, prefix, name, suffix);
    return oss.str();
}
```

Reply:
Yes it is identical except for the return.
Good idea to reuse `print`.